packages/wasm/src/keys.ts

```diff
@@ -28,6 +32,23 @@ export const getEphemeralByIndex = (fullViewingKey: string, index: number): Addr
   return new Address({ inner: uintArray });
 };
 
+export const getIndexByAddress = (fullViewingKey: string, address: string): AddressIndex => {
+  const res = validateSchema(
+    z.object({
+      account: z.number().optional(),
+      randomizer: z.string(),
+    }),
+    is_controlled_address(fullViewingKey, address),
+  );
+
+  const uintArray = base64ToUint8Array(res.randomizer);
+
+  return new AddressIndex({
+    account: res.account ?? 0,
+    randomizer: uintArray,
+  });
```

Comment:
I think we can do AddressIndex.fromJson(res) right?

Reply:
ok, but you use```
 new AddressIndex({
    account: res.account ?? 0,
    randomizer: uintArray,
  });
``` too
<img width="526" alt="Знімок екрана 2023-11-29 о 21 05 09" src="https://github.com/penumbra-zone/web/assets/52686175/3a2faa93-e9d2-4131-836a-bb9c91a31a81">

Reply:
ok, but you use 
```
new AddressIndex({
    account: res.account ?? 0,
    randomizer: uintArray,
  })
```

Reply:
But we don't need to manually deserialize it if the `fromJson` function is given to us

Reply:
```
 return new AddressIndex().fromJson({
    account: res.account ?? 0,
    randomizer: res.randomizer,
  });
```
this work, AddressIndex.fromJson(res) get types warnings, probably because account can be undefined